pkg/setting/setting_unified_alerting.go

```diff
@@ -150,7 +150,7 @@ func (cfg *Cfg) readUnifiedAlertingEnabledSetting(section *ini.Section) (*bool,
 	if !hasEnabled {
 		// TODO: Remove in Grafana v10
 		if cfg.IsFeatureToggleEnabled("ngalert") {
-			cfg.Logger.Warn("ngalert feature flag is deprecated: use unified alerting enabled setting instead")
+			cfg.Logger.Warn("Ngalert feature flag is deprecated: use unified alerting enabled setting instead")
```

Comment:
this is obsolete log and needs rephrasing: there is no more such a feature flag: it's a [setting](https://github.com/grafana/grafana/blob/a40302760800aef0c31511ff5830f329ac6c3b4b/conf/defaults.ini#L1033) now